### PR TITLE
Docs > Explain `True()` and `False()` predicates

### PR DESCRIPTION
**Pull request by ruudk**
Please update the docs when this is merged.

### CODE REVIEW

File: docs/reference/predicates.md [hunk 1]
```diff
@@ -152,6 +152,28 @@ route1: Path("/test") && Weight(100) -> "http://www.zalando.de";
 route2: Path("/test") && True() && True() -> "http://www.zalando.de";
 ```
 
+## True
+
+Does always match. Before `Weight` predicate existed this was used to give a route more weight.
+
+Example where `route2` is has more weight.
```
Comment (hjacobs) [inline docs/reference/predicates.md:159]: "is has" -> "has"

Comment (ruudk) [reply docs/reference/predicates.md]: Oops, fixed. Please use Github suggestions next time to 🙏

**Comment by szuecs** [comment]
@ruudk, gh suggestions break dco 🙁, so we can’t use it

**Comment by ruudk** [comment]
Argh damn dco :p

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**